customer_service/lib/ErrorReporter/Errors.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Lib\ErrorReporter;
+
+use Lib\ErrorReporter\ErrorNode;
+
+class Errors implements ErrorNode
+{
+    public $source;
+    public $previousNode;
+    public $nextNode;
+    public $errorMessage;
+    public $errorType;
+    public $userResponse;
+
+    public function setPreviousNode($errorNode)
+    {
+        $this->previousNode = $errorNode;
+    }
+
+    public function getPreviousNode()
+    {
+        return $this->previousNode;
+    }
+
+    public function setNextNode($errorNode)
+    {
+        $this->nextNode = $errorNode;
+    }
+
+    public function getNextNode()
+    {
+        if ($this->nextNode) {
+            return $this->nextNode;
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method getNextNode uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)